README.md

```diff
@@ -5,6 +5,11 @@
 ## Description
 > The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
+Demo view                                |  static view
+:---------------------------------------:|:---------------------------------------:
+![](./public/Animation.gif)              |  ![](./public/screenshot.PNG)
+
+
 ## Built With
 
```

Comment:
- [ ] In your `## Procedure to clone project` section, instead of saying this `open with live server`, I will recommend writing something like `Open the live server with 'npm run start'...`, this way you give clearly instructions of all the steps, think that your project could be seen for someone with no coding knowledge 😄

Reply:
thanks for the feedback, updated.